biochemist-python/chapters/Linear_Regression.ipynb

```diff
@@ -0,0 +1,752 @@
+    "\n",
+    "In this module, we will explore linear regression in Jupyter notebooks using Python. Please keep in mind - this is just a beginning. If you take a course in data science, you are likely to encounter a much deeper look at linear regression where you explore the relationships among many variables in a dataset.\n",
+    "\n",
+    "We're going to build on the previous lesson, using the pandas library to import the data for this lesson. Then, we will perform the linear regression using the SciPy library. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Libraries you will need\n",
+    "\n",
+    "In previous lessons, we have used `os`, `numpy`, and `pandas`. In this lesson, we will add the SciPy library, a collection of numerical tools for solving mathematical, scientific, technical and engineering problems (from the [Guru99 Python SciPy Tutorial](https://www.guru99.com/scipy-tutorial.html). Specific subsets of SciPy are useful for linear algebra (scipy.linalg), optimization and fit (scipy.optimize), statistics and random numbers (scipy.stats), and numerical integration (scipy.integrate).  \n",
+    "\n",
+    "We will use the dot notation we introduced earlier to access the functions in these libraries. I have expanded our table of libraries to help you keep track of your tools. Please note that the abbreviations listed are just the most common. You can actually define any abbreviation you like, but it is best to use the conventional abbreviations. This will help future coders (including yourself six months from now) as they work with the code. \n",
+    "\n",
+    "| Library | Uses | Abbreviation |\n",
+    "| :------- | :----: | :------------: |\n",
+    "| os | file management in operating systems | os |\n",
+    "| numpy | calculations | np  | \n",
+    "| pandas | data management | pd |\n",
+    "| scipy | calculations and statistics | sc or sp | \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Stages of this module\n",
+    "1. Importing the correct libraries\n",
+    "1. Importing data with pandas\n",
+    "1. Running simple linear regression to print out the desired statistics\n",
+    "1. Calculating protein sample concentrations using the linear regression results.\n",
+    "\n",
+    "The first part of the module is modeled after an exercise in Charlie Weiss's excellent online textbook, *Scientific Computing for Chemists*, which you can find on his GitHub site, [SciCompforChemists](https://github.com/weisscharlesj/SciCompforChemists)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the libraries you need\n",
+    "import os\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import scipy as sp"
```

Comment:
I noticed that we don't actually use this import in the notebook. Later we do `from scipy import stats`, but we don't actually use `sp` at all. I suggest changing the import at the start of the notebook to `from scipy import stats`, or not importing scipy until you are ready to do the fit.

Reply:
I added from scipy import stats on line 89. Then I deleted from scipy import stats from the cell where the regression is first called.